packages/wallet-sdk/src/usernames.ts

```diff
@@ -0,0 +1,18 @@
+import { fetchPrivate } from '@stacks/common';
+import { StacksNetwork } from '@stacks/network';
+
+export const fetchFirstName = async (
```

Comment:
Should this be moved to bns package?

Reply:
That works for me if @kyranjamie and @aulneau are in favor here.